target/linux/ramips/dts/mt7628an_wavlink_wl-wn577a2.dts

```diff
@@ -36,6 +36,16 @@
 	leds {
 		compatible = "gpio-leds";
 
+		led_lan: lan {
```

Comment:
you can drop the DT label (led_lan), you don't use it for anything. Same for led_wan.

Reply:
Is that mandatory ? I'd like to stick with the labels since the wps has one as well.

Reply:
Well, the idea is to only add a label when it is used. The led_wps label is actually used for referencing that led in aliases.

On the other hand, they won't hurt. So, you may as well keep them, but be prepared for somebody else to remove them anyway during a later cleanup.